### PR TITLE
Consolidate loadsdk debug statements

### PR DESCRIPTION
**Pull request by hamzy**
Merge all of the debug output statements into one defer code block.

### CODE REVIEW

File: pkg/destroy/powervs/powervs.go [hunk 1]
```diff
@@ -429,41 +452,31 @@ func (o *ClusterUninstaller) loadSDKServices() error {
 		return fmt.Errorf("loadSDKServices: bxsession.New: %v", err)
 	}
```
Comment (r4f4) [inline pkg/destroy/powervs/powervs.go:453]:
```suggestion
	}
	o.Logger.Debugf("loadSDKServices: bxSession = %v", bxSession)
```
I'm curious: why logging it once here is not enough?

**Review by r4f4 — COMMENTED** [review]
/approve

**Review by clnperez — APPROVED** [review]
/lgtm
thanks!

**Comment by r4f4** [comment]
/override ci/prow/golint
I ran the linter locally with a longer timeout, no problems reported.

**Pull request merged**